apm-agent-plugins/apm-es-restclient-plugin/apm-es-restclient-plugin-6_4/src/main/java/co/elastic/apm/agent/esrestclient/v6_4/ElasticsearchClientSyncInstrumentation.java

```diff
@@ -60,7 +60,7 @@ public static class ElasticsearchRestClientSyncAdvice {
         @Nullable
         @Advice.OnMethodEnter(suppress = Throwable.class, inline = false)
         public static Object onBeforeExecute(@Advice.Argument(0) Request request) {
-            return helper.createClientSpan(request.getMethod(), request.getEndpoint(), request.getEntity());
+            return helper.createClientSpan(request.getMethod(), request.getEndpoint(), request.getEntity(), false);
```

Comment:
shouldn't this be `true`?

Reply:
Sorry, I fixed it.

Reply:
shouldn't there have been a test failing from the wrong vaue? Means we're missing tests

Reply:
I added condition that checking `sync attribute` with sync requests.

Reply:
Now, if we set wrong value on instrumentation describen above we will get failure testss.